src/Caching/Impl/Models/FunctionAttributes.cs

```diff
@@ -15,16 +15,12 @@
 
 using System;
 
-namespace Microsoft.Python.Analysis.Types {
-    public interface IPythonFile {
```

Comment:
So this interface isn't needed any longer?

Reply:
It was never used anywhere, doesn't seem to provide any value.

Reply:
Ok